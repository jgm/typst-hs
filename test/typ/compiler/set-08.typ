#set page(margin: 1mm-0mm)
foo
