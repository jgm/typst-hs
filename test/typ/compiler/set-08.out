--- parse tree ---
[ Code
    "typ/compiler/set-08.typ"
    ( line 1 , column 2 )
    (Let
       (BasicBind (Just (Identifier "test")))
       (FuncExpr
          [ NormalParam (Identifier "x") , NormalParam (Identifier "y") ]
          (Block
             (CodeBlock
                [ If
                    [ ( Equals (Ident (Identifier "x")) (Ident (Identifier "y"))
                      , Block (Content [ Text "\9989" ])
                      )
                    , ( Literal (Boolean True)
                      , Block
                          (Content
                             [ Text "\10060"
                             , Text "("
                             , Code
                                 "typ/compiler/set-08.typ"
                                 ( line 1 , column 47 )
                                 (FuncCall
                                    (Ident (Identifier "repr"))
                                    [ NormalArg (Ident (Identifier "x")) ])
                             , Space
                             , Text "/"
                             , Text "="
                             , Space
                             , Code
                                 "typ/compiler/set-08.typ"
                                 ( line 1 , column 59 )
                                 (FuncCall
                                    (Ident (Identifier "repr"))
                                    [ NormalArg (Ident (Identifier "y")) ])
                             , Text ")"
                             ])
                      )
                    ]
                ]))))
, SoftBreak
, Code
    "typ/compiler/set-08.typ"
    ( line 2 , column 2 )
    (Set
       (Ident (Identifier "page"))
       [ KeyValArg
           (Identifier "margin")
           (Minus (Literal (Numeric 1.0 Mm)) (Literal (Numeric 0.0 Mm)))
       ])
, SoftBreak
, Text "foo"
, ParBreak
]
--- evaluated ---
document(body: { text(body: [
]), 
                 text(body: [
foo]), 
                 parbreak() })
